From 76c40d80fed704f6c3dfaf2e41f44554b7d28797 Mon Sep 17 00:00:00 2001
From: Timothy D Witham <twitham@sbcglobal.net>
Date: Sat, 9 Sep 2023 01:58:09 -0500
Subject: [PATCH] MythMusic: fix bug that switching from Gallery to Tree broke
 repaints

upon escape back to the playlist.  I don't know what these disconnects
were for, but removing them restores updates to the parent window.
---
 mythplugins/mythmusic/mythmusic/musiccommon.cpp | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/mythplugins/mythmusic/mythmusic/musiccommon.cpp b/mythplugins/mythmusic/mythmusic/musiccommon.cpp
index 8693efadda3..8d4507b71d8 100644
--- a/mythplugins/mythmusic/mythmusic/musiccommon.cpp
+++ b/mythplugins/mythmusic/mythmusic/musiccommon.cpp
@@ -478,10 +478,7 @@ void MusicCommon::switchView(MusicView view)
                 delete pleview;
 
             if (oldView)
-            {
-                disconnect(this, &MythScreenType::Exiting, nullptr, nullptr);
                 Close();
-            }
 
             break;
         }
@@ -508,10 +505,7 @@ void MusicCommon::switchView(MusicView view)
                 delete pleview;
 
             if (oldView)
-            {
-                disconnect(this, &MythScreenType::Exiting, nullptr, nullptr);
                 Close();
-            }
 
             break;
         }
